docs/src/main/asciidoc/native.adoc

```diff
@@ -0,0 +1,107 @@
+== Native Image Support (Experimental)
+
+https://www.graalvm.org/docs/[GraalVM Native Image] allows you to ahead-of-time compile Java code to a standalone executable.
+GraalVM native images are supported in Spring Boot since 3.0 and Spring Framework 6.0.
+
+WARNING: Native Image Support for Spring Framework on Google Cloud is **Experimental**.
+Compatibility with native image compilation are available "as is" and might have limited support.
```

Comment:
> "Spring Framework on Google Cloud" --> "Spring Cloud GCP"

We explicitly remove usage of "Spring Cloud GCP" in favor of  "Spring Framework on Google Cloud" because GCP is a deprecated name.

Addressed other 2 in 348f6bfc6